apis/Google.Cloud.Storage.V1/docs/history.md

```diff
@@ -1,5 +1,9 @@
 # Version history
 
+# Version 3.4.0-beta01, released 2020-09-30
```

Comment:
Are there *any* other changes, e.g. to dependencies? If not, we could make this a patch release. It might be simplest to keep it as a minor release but also bump the Google.Api.Storage.v1 dependency as there may well have been changes there.

Reply:
Updated dependencies on the first commit and included it in history.md on the second commit.